API/c/Inc/analog_in.h

```diff
@@ -0,0 +1,11 @@
+#ifndef AnalogIn
+#define AnalogIn
+#include "pin_defs.h"
+
+void adc_init(adc_pin_name pin);
```

Comment:
There should probably be a deinit function. See http://gitlab.kierenblack.nz/kieren/Arduino_Core_STM32/blob/2fbfed07eb07fc74844d580f03bb7d453d23e573/cores/arduino/stm32/analog.c

Reply:
Yup Yup ^^^ @joeysteele2001